agent/src/agent.php

```diff
@@ -173,4 +182,41 @@
 
 $ingestDetails->hydrate();
 
+$signatureCheckTimer = $loop->addPeriodicTimer(60, static function () use ($loop, &$signatureCheckTimer, $ingest, $error, $info, $basePath, $expectedSignature) {
+    $signature = file_get_contents($basePath.'/signature.txt');
+
+    if ($signature === false) {
+        $error('Unable to read signature');
+
+        return;
+    }
+
+    if ($signature !== $expectedSignature) {
+        if ($signatureCheckTimer instanceof \React\EventLoop\TimerInterface) {
```

Comment:
We should be able to remove this conditional. This will always be an instance of the timer at runtime. We can force PHP tooling to know that by adding a `@var` comment at the top of the closure.